pkg/users/users.js

```diff
@@ -82,7 +82,7 @@ function AccountsPage() {
         const handleShadow = cockpit.file("/etc/shadow", { superuser: "try" });
         handleShadow.watch(() => debouncedGetLogins(), { read: false });
 
-        const handleLogindef = cockpit.file("/etc/login.defs", { superuser: true });
+        const handleLogindef = cockpit.file("/etc/login.defs", { superuser: "try" });
```

Comment:
This is a behaviour change, but /etc/login.defs is world-readable and only used for watching -- why would it need superuser in the first place? But anyway, for the purpose of this PR it's fine.

Reply:
I rather we clean it up then and leave superuser out.

Reply:
That's done now :)